src/plugins/blocksmith/components/blocksmith/Blocksmith.jsx

```diff
@@ -1535,22 +1497,8 @@ class Blocksmith extends React.Component {
           return constants.DOUBLE_ENTER_ON_LIST;
         }
       }
-    } else if (
-      /^[cx]$/.test(e.key)
-      && ((e.metaKey && isMacOS()) || (e.ctrlKey && isWindows()))
-      && isAtomicBlockSelected(editorState)
-    ) {
-      if (e.key === 'c') {
-        selection.capture(editorState);
-        copySelectedBlocksToClipboard(editorState);
-        selection.restore();
```

Comment:
@tbone1020 i think this selection util is also no longer necessary

Reply:
done